css/properties/column-width.json

```diff
@@ -0,0 +1,177 @@
+            },
+            "ie_mobile": {
+              "version_added": true
+            },
+            "opera": {
+              "version_added": "11.10"
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "prefix": "-webkit-",
+              "version_added": "3"
+            },
+            "safari_ios": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "intrinsic_sizes": {
+          "__compat": {
+            "description": "Intrinsic sizes",
+            "support": {
+              "webview_android": {
+                "version_added": false
+              },
+              "chrome": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "ie": {
+                "version_added": false
+              },
+              "ie_mobile": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": false
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": {
+                "version_added": false
+              }
+            }
+          }
+        },
+        "on_display_table_caption": {
+          "__compat": {
+            "description": "On <code>display: table-caption</code>",
+            "support": {
+              "webview_android": {
+                "version_added": true
+              },
+              "chrome": {
+                "version_added": true
+              },
+              "chrome_android": {
+                "version_added": "50"
+              },
+              "edge": {
+                "version_added": true
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": "37"
+              },
+              "firefox_android": {
+                "version_added": "37"
+              },
+              "ie": {
+                "version_added": true
+              },
+              "ie_mobile": {
+                "version_added": true
+              },
+              "opera": {
+                "version_added": true
+              },
+              "opera_android": {
+                "version_added": true
+              },
+              "safari": {
+                "version_added": true
+              },
+              "safari_ios": {
+                "version_added": true
+              }
+            }
```

Comment:
Status is missing too. Is this a standard behaviour?

Reply:
Like above, I'm not sure. I looked over the standards documents link from both the `column-width` and `display` property pages and couldn't find any mention of interaction between the two. Again, I don't know what behavior this refers to.

Reply:
It is.

Reply:
OK, added status statement.